src/containers/Category.js

```diff
@@ -11,14 +12,15 @@ export const Category = () => {
     city,
     category
   );
+  const businessList = businesses.sort((a, b) => (a.name > b.name ? 1 : -1));
 
-  const BusinessesList = businesses
-    .sort((a, b) => (a.name > b.name ? 1 : -1))
-    .map((business) => (
-      <Box m={4}>
-        <BusinessInfoCard business={business} />
-      </Box>
-    ));
-
-  return <Page>{BusinessesList}</Page>;
+  return (
+    <DiscoveryPage title={categories[category].name} subtitle={categories[category].description}>
+      <Grid container spacing={8}>
```

Comment:
We should specify `direction='column'` here. Or give every child element a proper width. Otherwise, the children won't necessarily be the same width, and then the second column doesn't line up.

![image](https://user-images.githubusercontent.com/16708427/85335013-e866b180-b4a1-11ea-9fdf-2f00ce40bca1.png)

Reply:
Also, the second column is awkwardly far away from the first column on the desktop view.